### PR TITLE
fix(k8s): don't store full version object with test+task results

### PR DESCRIPTION
**Pull request by edvald**
This addresses issues where the list of files, contained in the module
version object, would push the size of the result ConfigMap over 1MB.

### CODE REVIEW

File: garden-service/src/plugins/kubernetes/task-results.ts [hunk 1]
```diff
@@ -83,6 +90,8 @@ export async function storeTaskResult(
       [gardenAnnotationKey("moduleVersion")]: module.version.versionString,
       [gardenAnnotationKey("version")]: taskVersion.versionString,
     },
-    data: result,
+    data: {
```
Comment (10ko) [inline garden-service/src/plugins/kubernetes/task-results.ts]: Why `data: {}`?

**Comment by 10ko** [comment]
Tested and fixes my issues, even when running tests *with* `node_modules` outside the container and *without* a `.gardenignore`. 
Aside the question above, LGTM. Thanks!

File: garden-service/src/plugins/kubernetes/task-results.ts (hunk 1 above)
Comment (edvald) [reply garden-service/src/plugins/kubernetes/task-results.ts]: That's a mistake, I'll fix it.

**Comment by edvald** [comment]
@10ko fixed that line. Can you give a stamp of approval? :)

**Pull request merged**